### PR TITLE
Chore: inline Variables that immediately Returned

### PR DESCRIPTION
**Pull request by yezz123**
- I just fix some issues related to code.
- Refactor some Parts of the code.

### CODE REVIEW

**Review by auvipy — CHANGES_REQUESTED** [review]
can you please update tests to see this changes wont regress?

**Comment by yezz123** [comment]
> can you please update tests to see this changes wont regress?

It's not a new feature to resolve in the tests, cause this is just a simple way to refactor some parts of code, I didn't add any function that why 😀

**Comment by auvipy** [comment]
I understand but still the changes are significant so you can add some additional test on existing tests only

**Comment by yezz123** [comment]
> I understand but still, the changes are significant so you can add some additional tests on existing tests only

I don't see where we can add this test, cause this is just some simple changes that why 🤔

**Comment by auvipy** [comment]
https://github.com/jpadilla/pyjwt/tree/master/tests tests for related modules can be found here

**Comment by beliaev-maksim** [comment]
@auvipy,
I just review changes. It is purely refactoring, no functionality was changed, you do not need additional tests

**Review by beliaev-maksim — APPROVED** [review]
pure refactoring, looks good

**Comment by dajiaji** [comment]
+1

File: jwt/api_jws.py [hunk 1]
```diff
@@ -90,9 +90,6 @@ def encode(
         if headers and "alg" in headers and headers["alg"]:
             algorithm = headers["alg"]
 
-        if algorithm not in self._valid_algs:
```
Comment (auvipy) [inline jwt/api_jws.py:93]: may I know why this was removed?

**Review by auvipy — APPROVED** [review]

File: jwt/api_jws.py (hunk 1 above)
Comment (beliaev-maksim) [reply jwt/api_jws.py:93]: Because it just `pass` statement, it has no influence except time spent for if condition check

**Pull request merged**